starter-code/src/bubble-sort.js

```diff
@@ -1,12 +1,12 @@
 function bubbleSort(array) {
-    var length = array.length;
-    for (var i = (length - 1); i > 0; i--) {
+    const length = array.length;
+    for (let i = (length - 1); i > 0; i--) {
         // Number of passes
-        for (var j = (length - i); j > 0; j--) {
+        for (let j = (length - i); j > 0; j--) {
             // Compare the adjacent positions
             if (array[j] < array[j - 1]) {
                 // Swap the numbers
-                var tmp = array[j];
+                let tmp = array[j];
```

Comment:
You can use `const` here because the variable is never reassigned inside its block.